src/common/libflux/conf.c

```diff
@@ -55,6 +55,7 @@ static struct config default_config[] = {
     { "keydir",         NULL,                       INTREE_KEYDIR },
     { "no_docs_path",   INSTALLED_NO_DOCS_PATH,     INTREE_NO_DOCS_PATH },
     { "rundir",         INSTALLED_RUNDIR,           NULL },
+    { "bindir",         NULL,                       INTREE_BINDIR },
```

Comment:
One question, not really a review comment: could we also set the INSTALLED_BINDIR here just for consistency? I was initially confused since `bindir = ${exec_prefix}/bin` is a very common variable used in autoconf.

Reply:
Sure, I'll fix that.